src/backend/joanie/core/serializers/client.py

```diff
@@ -640,12 +649,14 @@ class ProductSerializer(serializers.ModelSerializer):
     target_courses = ProductTargetCourseRelationSerializer(
         read_only=True, many=True, source="target_course_relations"
     )
+    contract_definition = ContractDefinitionSerializer(read_only=True)
```

Comment:
Until we really need it, I think a SlugRelatedField is enough to just return the id. WDYT ?